api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -92,6 +98,48 @@ public synchronized Order saveOrder(Order order, OrderContext orderContext) thro
 	}
 
 	/**
+	 * @see org.openmrs.api.OrderService#saveOrders(Set)
+	 */
+	public synchronized Set<Order> saveOrders(Set<Order> orders) {
```

Comment:
In other words the above method takes care of saving both the orders which are ordered independently and the orders which have an orderGroup.